committee/recordtableoperator.go

```diff
@@ -304,6 +304,17 @@ func (arch *recordTableOperator) identicalTo(height uint64, sdb *sql.DB, tx *sql
 	}
 }
 
+func (arch *recordTableOperator) lastHeight(height uint64, tx *sql.Tx) (uint64, error) {
+	var val int64
+	err := tx.QueryRow(arch.lastHeightQuery, util.Uint64ToInt64(height)).Scan(&val)
+	switch err {
+	case nil:
+		return uint64(val), nil
+	default:
```

Comment:
for empty table, there will be a query error returned by QueryRow. In this case, we need to have special handling.